src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -6,8 +6,8 @@
     <RootNamespace>Microsoft.Azure.WebJobs.Extensions.DurableTask</RootNamespace>
     <MajorVersion>2</MajorVersion>
     <MinorVersion>10</MinorVersion>
-    <PatchVersion>0</PatchVersion>
-    <Version>$(MajorVersion).$(MinorVersion).$(PatchVersion)</Version>
+    <PatchVersion>1</PatchVersion>
+    <Version>$(MajorVersion).$(MinorVersion).$(PatchVersion)-preview</Version>
```

Comment:
Use a more specific suffix for private packages. `preview` is reserved for things we intend to make publicly available. Also recommend adding a `.1` at the end so you can rev that number if you need to add any more changes (and not need to rev patch version).

For example, entities private builds are `-entities-preview.1` and now will be `-entities.preview.2`